frontend/locales/en/nav.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
@rebeccaalpert We'll need to make sure the checked in i18n files are current in CI so we don't forget to run `yarn i18n`. I'd probably just update test-frontend.sh to run `yarn i18n` and check if there is a git diff. Something like this (untested)

```bash
GIT_STATUS="$( git status --short --untracked-files -- frontend/locales )"
if [[ -n "$GIT_STATUS" ]]; then
  echo "i18n files are not up to date. Run `yarn i18n` to re-generate."
  git diff
  exit 1
fi
```